modules/apps/29-fee/keeper/events_test.go

```diff
@@ -146,19 +146,22 @@ func (suite *KeeperTestSuite) TestDistributeFeeEvent() {
 	events := res.GetEvents()
 	expectedEvents := sdk.Events{
 		sdk.NewEvent(
-			types.EventTypeDistributeFee,
-			sdk.NewAttribute(types.AttributeKeyReceiver, suite.chainA.SenderAccount.GetAddress().String()),
-			sdk.NewAttribute(types.AttributeKeyFee, defaultRecvFee.String()),
+			"distribute_fee",
```

Comment:
I refactored this and put the hard-coded strings, so that in case the `const` keys change we detect it in the test.